srcpkgs/aravis/template

```diff
@@ -18,7 +19,7 @@ checksum=d70b125666b23ca4c0f8986fa0786a3d2b9efb7a56b558b703083cdfaa793f4e
 make_check=ci-skip
 
 if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" glib-devel prelink-cross qemu-user"
+	hostmakedepends+=" glib-devel prelink-cross"
 fi
```

Comment:
Sound likes `gir` build helper with `glib-devel` in `hostmakedepends` and `libglib-devel` in `makedepends`